app.py

```diff
@@ -0,0 +1,197 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from flask import Flask, jsonify, request
+
+import base64
+import os
+import re
+import json
+import webbrowser
+import traceback
+import datetime
+import cv2
+import easyocr
+import time
+import torch
+import ipdb
+import logging
+import threading
+import requests
+
+from WebHandler.scrappers.highcourts import get_highcourt_cases_by_name, get_no_of_cases
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+sh = logging.StreamHandler()
+sh.setLevel(logging.DEBUG)
+logger.addHandler(sh)
+path = os.environ.get('DOWNLOAD_PATH')
+
+
+def create_driver(__location__):
+    options = Options()
+    DRIVER_PATH = os.environ.get('DRIVER_PATH')
+    options.add_argument("--disable-extensions")
+    options.add_argument("--disable-infobars")
+    options.add_argument("--headless")
+    options.add_argument("--disable-dev-shm-usage")
+    options.add_argument("--no-sandbox")
+    options.add_argument("--window-size=1700x800")
+    prefs = {
+        "browser.helperApps.neverAsk.saveToDisk": "application/octet-stream;application/vnd.ms-excel;text/html;application/pdf",
+        "pdfjs.disabled": True,
+        "print.always_print_silent": True,
+        "print.show_print_progress": False,
+        "browser.download.show_plugins_in_list": False,
+        "browser.download.folderList": 2,
+        # Change default directory for downloads
+        "download.default_directory": __location__,
+        "download.prompt_for_download": False,  # To auto download the file
+        "download.directory_upgrade": True,
+        "plugins.always_open_pdf_externally": True
+    }
+
+    options.add_experimental_option("prefs", prefs)
+    driver = webdriver.Chrome(DRIVER_PATH, chrome_options=options)
+    driver.maximize_window()
+    return driver
+
+
+def fire_and_forget(f):
+    def wrapped():
+        threading.Thread(target=f).start()
+
+    return wrapped
+
+
+app = Flask(__name__)
+
+
+@app.route("/", methods=["POST"])
+def main():
+    permitted_cases = int(os.environ.get('CASES_PER_ITERATION'))
```

Comment:
add a default incase that env is not present 
os.environ.get('CASES_PER_ITERATION', 50)